PHP/Toolset.md

```diff
@@ -70,6 +90,11 @@ Event-driven programming with ReactPHP
 
 Using Supervisord for php scripts
 
+Aggregation profiling using [Live Profiler(lifeprof)](https://github.com/badoo/liveprof), [New Relic](https://newrelic.com/), [Pinba](http://pinba.org/)
```

Comment:
Last commit on Pinba is 4 years ago. Is it still relevant?

Reply:
Yeah, good catch. I have added it because have seen a lot of articles about using it some time ago. The latest was in 2019 https://systemadminspro.com/statistics-and-monitoring-of-php-scripts-in-real-time/ 
But I indeed need to reconsider adding it.

Reply:
Replaced with Pinba2